Makefile

```diff
@@ -25,4 +25,9 @@ VECTORFIGURES =
 # Additional files to distribute (e.g., CSS, schema files, examples...)
 AUX_FILES =
 
-include ivoatex/Makefile
+-include ivoatex/Makefile
+
+ivoatex/Makefile:
+	@echo "*** ivoatex submodule not found.  Initialising submodules."
+	@echo
+	git submodule update --init
```

Comment:
It is a good idea to do this, but can we check that this works in `ivoatex-docker` before this is merged into the main branch.
See https://github.com/ivoa/ivoatex-docker/issues/3

Reply:
Does explicitly adding a new dependency to the build process fall under the heading 'minor technical updates' ?

Reply:
We should probably check that this works with the automated build process for preview documents.